examples/vertx-opentracing/src/main/java/org/hawkular/apm/examples/vertx/opentracing/OrderLog.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.apm.examples.vertx.opentracing;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.propagation.Format;
+import io.vertx.core.eventbus.EventBus;
+import io.vertx.core.eventbus.Message;
+import io.vertx.core.eventbus.MessageConsumer;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * @author gbrown
+ */
+public class OrderLog {
+
+    private Map<String, JsonArray> orders = new HashMap<>();
+    private MessageConsumer<JsonObject> ordersConfirmedConsumer;
+    private MessageConsumer<JsonObject> getOrdersConsumer;
+
+    private OrderLog(EventBus eb, Tracer tracer) {
+        init(eb, tracer);
+    }
+
+    public static OrderLog create(EventBus eb, Tracer tracer) {
+        return new OrderLog(eb, tracer);
+    }
+
+    protected void init(EventBus eb, Tracer tracer) {
```

Comment:
Could you break this method to two? One for `ordersConfirmedConsumer` and second `getOrdersConsumer`.